### ISSUE TITLE
Suggestion: File actions after listing a directory

### ISSUE THREAD

**Issue by gsabater** [mkloubert/vscode-deploy-reloaded#68]
### Description

It would be nice if I could list a remote folder and then perform actions on remote files.
For example, i want to list my images folder, and there is one image that i have removed locally, and i would like to remove remotely afterwards. Clicking on the item edits the item, and now, there is no way to delete the remote file.

### Actual behavior

Edits the remote file on click

### Expected behavior

Show a menu with multiple actions: edit, remote, rename, download...

**Comment by mkloubert** [comment]
@gsabater 

Have you tried my extension [vscode-remote-workspace](https://marketplace.visualstudio.com/items?itemName=mkloubert.vscode-remote-workspace)?

I think it provides all file actions you need.

**Comment by gsabater** [comment]
@mkloubert 
Yes, vscode-remote-workspace has all that features, but our team can't work directly on a remote server, we work in local and then we upload the files on the server. 

But sometimes we need to change a file name only on the server or perform any action on the server because the file is not in local, and it would be awesome if we could do that with a single extension.

Thanks for your time, and sorry for the issue.
Feel free to close it if you don't feel like adding the features to deploy-reloaded.
